torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -1204,6 +1204,7 @@ RegisterOperators reg(
            TypePtr value_type = static_cast<const DictType*>(output_type.get())->getValueType();
            return [=](Stack& stack) {
              auto vals = c10::impl::GenericDict(key_type, value_type);
+             vals.reserve(stack.size() / 2);
```

Comment:
How is `stack`'s size related to the size of `vals`

Reply:
```
             for (size_t i = 0; i < num_inputs; i += 2) {
               auto val = pop(stack);
               auto key = pop(stack);
               vals.insert_or_assign(std::move(key), std::move(val));
             }
```

Reply:
Shouldn't it be `num_inputs * 2` then?

Reply:
no because each iteration the dict size increases by one